docs/astro/files/operators/get_file_list.rst

```diff
@@ -8,6 +8,10 @@ When to use the ``get_file_list`` operator
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 We can use ``get_file_list`` when you want a list file available in storage based on storage path pattern and Airflow connection. This might useful when you want to create dynamic task on the response of this operator.
 
+The supported filesystem are ``Local, S3, GCS, and HTTP``.
```

Comment:
Don't we have a rst file containing all the supported filesystems? like we have for supported databases

Reply:
Not sure. I can see we have a supported technologies section in README https://github.com/astronomer/astro-sdk#supported-technologies. Referring to this here would misguide.  Any thought on how we can improve README and refer here?

Reply:
@pankajastro you can use - https://github.com/astronomer/astro-sdk/blob/main/docs/supported_file.rst

Reply:
fixed 9b436a9190564e4067c3f643859d5834c83223fb